src/v2/Apps/Fair/Components/FairExhibitorRail/FairExhibitorRailArtworks.tsx

```diff
@@ -7,25 +7,65 @@ import { FAIR_EXHIBITOR_IMAGE_HEIGHT } from "./FairExhibitorRail"
 import { FairExhibitorRailPlaceholder } from "./FairExhibitorRailPlaceholder"
 import { Carousel } from "v2/Components/Carousel"
 import FillwidthItem from "v2/Components/Artwork/FillwidthItem"
-import { ContextModule } from "@artsy/cohesion"
+import {
+  ActionType,
+  ClickedArtworkGroup,
+  ContextModule,
+  OwnerType,
+} from "@artsy/cohesion"
+import { useTracking } from "react-tracking"
 
 export interface FairExhibitorRailArtworksProps {
   show: FairExhibitorRailArtworks_show
+  fairID: string
+  fairSlug: string
 }
 
 const FairExhibitorRailArtworks: React.FC<FairExhibitorRailArtworksProps> = ({
   show,
+  fairID,
+  fairSlug,
 }) => {
+  const tracking = useTracking()
+
+  const clickedFairArtworkData = ({
+    artworkID,
+    artworkSlug,
+    carouselIndex,
+  }) => {
```

Comment:
It'd be better to specify the return type `ClickedArtworkGroup` rather than asserting with `as` since [that may ignore lead to some erroneous data](https://basarat.gitbook.io/typescript/type-system/type-assertion#assertion-considered-harmful)